task8-3/task8-3/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace task8_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите вещественное число а>1");
+            double a = double.Parse(Console.ReadLine());
+
+            double summa = 0;
+            int initialint = 2;
```

Comment:
Первое слагаемое упущено. Либо
```suggestion
            double summa = 1;
            int initialint = 2;
```
Либо
```suggestion
            double summa = 0;
            int initialint = 1;
```